python/src/main/python/drivers/glsl-to-spv-worker.py

```diff
@@ -451,30 +462,24 @@ def main():
     worker_info_file = 'worker_info.json'
     remove(worker_info_file)
 
-    if args.linux:
-        vkrun.dump_info_linux()
+    if is_android:
+        runspv.dump_info_android_legacy(wait_for_screen=not args.force)
```

Comment:
These lines would require that, if the user is not using the legacy worker option then the user must have both the legacy worker and amber ready-to-go. Should we perhaps not require this? E.g. only dump the info when using the legacy option?

Reply:
I have changed this to ignore any errors when not using the legacy worker.